webpack/scenes/Subscriptions/Manifest/SimpleContentAccess.js

```diff
@@ -11,6 +11,8 @@ const SimpleContentAccess = (props) => {
     isSimpleContentAccessEnabled,
     enableSimpleContentAccess,
     disableSimpleContentAccess,
+    infoMessage,
```

Comment:
i guess all changes here can be reverted?

Reply:
I kept the changes in `SimpleContentAccess.js` (namely, adding the `colWidths` and `infoMessage` props) because I still think they make the component more flexible and reusable.  (also I think I'd have to revert it manually :confused:)   would you be open to keeping them?

Reply:
They do add that to the component, but I think there's a very high chance we'll never benefit from those changes so I'd prefer to see them removed. However, I'll defer to you!

If you did want to restore the file: `git reset upstream/master webpack/scenes/Subscriptions/Manifest/SimpleContentAccess.js`